tracing-attributes/src/attr.rs

```diff
@@ -108,8 +108,9 @@ impl Parse for InstrumentArgs {
                 if args.name.is_some() {
                     return Err(input.error("expected only a single `name` argument"));
                 }
-                let name = input.parse::<StrArg<kw::name>>()?.value;
-                args.name = Some(name);
+                let _ = input.parse::<kw::name>()?;
+                let _ = input.parse::<Token![=]>()?;
+                args.name = Some(input.parse()?);
```

Comment:
When reading this, it can be hard to understand why we parse 3 times in order to get the name. Could you leave a small comment to explain what the steps here are?

Reply:
Replaced with the `ExprArg` helper.